### ISSUE TITLE
Accessibility: the left menu does not announce collapsible sections

### ISSUE THREAD

**Issue by ghost** [opentofu/registry-ui#152]
The left menu does not properly contain the attributes for collapsible sections and therefore, does not announce with a screen reader if a section is open/closed, or when the opening is triggered.
